src/vue/common/IdleMessage.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div class="idle__message-wrapper">
+    <drawer
+      @close-drawer="redirectToSignin"
+      :is-shown.sync="isDetailsDrawerShown"
+    >
+      <template slot="heading">
+        {{ 'idle.form-heading' | globalize }}
+      </template>
+      <div>
+        <h2 class="idle__message-text">
+          {{ 'idle.notification-message' | globalize }}
+        </h2>
+      </div>
+    </drawer>
+  </div>
+</template>
+
+<script>
+import Drawer from '@/vue/common/Drawer'
+import { vuexTypes } from '@/vuex'
+import { mapMutations } from 'vuex'
+
+export default {
+  name: 'idle-message',
+  components: {
+    Drawer,
+  },
+
+  data: _ => ({
+    isDetailsDrawerShown: false,
+  }),
+
+  onIdle () {
+    this.isDetailsDrawerShown = true
+    this.clearState()
+  },
+
+  computed: {
+    email: vuexTypes.walletEmail,
+    typeI: vuexTypes.accountTypeI,
```

Comment:
What is this? Remove`typeI` and `email`. Why you removed the only mapGetters keyword, but not unused getters too?